### ISSUE TITLE
Missing Zip in OpenCombine

### ISSUE THREAD

**Issue by ManaliM27** [broadwaylamb/OpenCombine#203]
Is it possible to zip 2 publishers similar to combine

**Comment by MaxDesiatov** [comment]
Not until https://github.com/OpenCombine/OpenCombine/pull/109 is merged. Closing this as a duplicate of https://github.com/OpenCombine/OpenCombine/issues/1, this is where we track our progress for all of the operators.
